docs/modules/operation/pages/flows/classification-engine.adoc

```diff
@@ -90,31 +92,30 @@ Assume that you define the following rules:
 |===
 
 == Create a flows classification rule
+
 A flows classification rule must include at least a name and one definition for source/destination port, source/destination address, IP protocol, or exporter filter.
 
 By default, rules that a user creates appear in the "user-defined rules" group.
 You can <<rule-groups, create other groups>> as needed and assign rules to those groups.
 
 . Click the gears icon, and click *Manage Flow Classification*.
 . In the *User-defined Rules* tab, click the plus sign.
-. In the *Application Name* field, type a name for the rule.
+. Type a name for the rule in the *Application Name* field.
 . Specify a value in at least one of the following fields:
 +
-
 [cols="1,3"]
 |===
-
 | Source IP Address
 | The source IP address of the flow to match.
-May contain wildcards.
+May include wildcard characters (for example, asterisks (*)) to define a dynamic range of IP addresses that can be matched.
 
 | Source Port
 | The source port of the flow to match.
 May be a range or list of ports; for example, 80,8080,8980, or 8000-9000.
 
 | Destination IP Address
 | The destination IP address of the flow to match.
-May contain wildcards.
+May include wildcard characters (for example, asterisks (*)) to define a dynamic range of IP addresses that can be matched.
```

Comment:
Are there any other wildcard characters that we accept? If so, we should list them.

Reply:
Other wildcard characters are accepted in the Exporter Filter field, and its description already references the Filters API page with descriptions of the syntax. In this case, only asterisks are accepted, so no changes are necessary. :)